cli/schemas/config-file.v1.json

```diff
@@ -249,6 +249,10 @@
               "uniqueItems": true
             }
           }
+        },
+        "report": {
+          "type": "string",
```

Comment:
I think it might be better to use `enum` here with `default` set to `pretty`

Reply:
should be updated